packages/kbn-check-mappings-update-cli/current_fields.json

```diff
@@ -285,6 +285,7 @@
     "refreshInterval.pause",
```

Comment:
When I try to save a dashboard with a collapsabile section just added, I get:
<img width="414" alt="Screenshot 2025-05-29 at 12 13 26" src="https://github.com/user-attachments/assets/0c906cef-0cad-4717-a6d6-271d28d0f43c" />

> Content management client error: Invalid data. [panels.12.type]: expected value of type [string] but got [undefined]

Reply:
Do you have more thorough reproduction steps here? I am not seeing this:


https://github.com/user-attachments/assets/daea5e14-65f9-4a71-bcc7-4570d27fda88